test-suite/shared/src/test/require-jdk8/org/scalajs/testsuite/javalib/time/DayOfWeekTest.scala

```diff
@@ -0,0 +1,103 @@
+package org.scalajs.testsuite.javalib.time
+
+import java.time._
+import java.time.temporal.ChronoField
+
+import org.junit.Assert._
+import org.junit.Test
+import org.scalajs.testsuite.utils.AssertThrows._
+
+class DayOfWeekTest extends TemporalAccessorTest {
+  import DayOfWeek._
+
+  val samples = values.toSeq
+
+  def isSupported(field: ChronoField): Boolean =
+    field == ChronoField.DAY_OF_WEEK
+
+  @Test def should_respond_to_getValue(): Unit = {
```

Comment:
Could you remove the `should_respond_to_` from all the test method names. We are changing the format because with JUnit the names become a bit too long.